pkg/disruption/backend/recorder/shutdown.go

```diff
@@ -0,0 +1,64 @@
+package recorder
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/events"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/openshift/origin/pkg/disruption/backend"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+)
+
+func NewShutdownWindowRecorder(monitorRecorder backend.Recorder, eventRecorder events.EventRecorder, locator, name string) *shutdownWindowRecorder {
```

Comment:
Godoc would be great on this one but I'm curious why shutdown windows are being modeled / recorded within the disruption framework rather than outside, perhaps by postprocessing the intervals after the job run. (which are already recorded)